src/OutdatedCommand.ts

```diff
@@ -71,6 +73,10 @@ export class OutdatedCommand extends BaseCommand {
     description: "Format the output as JSON",
   })
 
+  markdown = Option.Boolean("--markdown", false, {
+    description: "Format the output as a markdown table",
+  })
```

Comment:
Instead, it might be better to support a new `format` flag that way there isn't the concurrent flag issue.

```bash
yarn outdated --format=json
yarn outdated --format=md
yarn outdated --json # deprecated
```